sklearn/cluster/tests/test_k_means.py

```diff
@@ -804,8 +750,8 @@ def test_k_means_function():
     assert cluster_centers.shape == (n_clusters, n_features)
     assert np.unique(labels).shape[0] == n_clusters
 
-    # check that the labels assignment are perfect (up to a permutation)
-    assert_allclose(v_measure_score(true_labels, labels), 1.0)
```

Comment:
For this one we don't expect the labels to match perfectly since the true labels are manually defined. We can only try to remove the v_measure_score when comparing 2 kmeans run that should be identical